src/subtask.ts

```diff
@@ -14,7 +14,12 @@ export class ULabelSubtask {
         active_id: string,
         annotation_mode: string,
         back_context: CanvasRenderingContext2D,
-        edit_candidate: unknown, // TODO: figure out what type this is
+        edit_candidate: {
+            annid: string,
+            access: ULabelAnnotation,
```

Comment:
```suggestion
            access: [,
```
This is actually wthat is referred to as an "access string" internally, and is different from the annotation access lists. Its purpose is to refer to the point within a spatial payload that is being edited, See `geometric_utils.ts` type `PointAccessObject`:

```ts
export type PointAccessObject = {
    "access": String | number, // Access string or number that acts as the index of the point in the original spatial payload
    "distance": number,
    "point": Point2D
}
```
and technically this isnt completely accurate, since for polygons it is actually `[number, number]` where the first number is the index of the polygon layer and the second number is the index of the point.